<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Conformers</class>
 <widget class="QWidget" name="Tab_Conformers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Molecule:</string>
     </property>
     <property name="buddy">
      <cstring>combo_mol</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="combo_mol">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="minimumContentsLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="9">
    <widget class="QTableWidget" name="table_conformers">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Conformer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Energy</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Probability</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Method:</string>
     </property>
     <property name="buddy">
      <cstring>combo_opt</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="combo_opt">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>Gaussian 03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ghemical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MMFF94</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MMFF94s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UFF</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Number of conformers:</string>
     </property>
     <property name="buddy">
      <cstring>spin_nConformers</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spin_nConformers">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="cb_allConformers">
     <property name="text">
      <string>&amp;All</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="push_generate">
     <property name="text">
      <string>&amp;Generate conformers</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_mol</tabstop>
  <tabstop>push_generate</tabstop>
  <tabstop>combo_opt</tabstop>
  <tabstop>spin_nConformers</tabstop>
  <tabstop>cb_allConformers</tabstop>
  <tabstop>table_conformers</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_allConformers</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_nConformers</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
